demo-edm/.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules/
 .tmp_edm
+SP 909 From Mars
```

Comment:
Is this .gitignore line necessary? If you want to keep these files around, I think the default sample path is `~/Library/Application\ Support/cybr/sample/` (cybr should be able to find them there if you move this directory there)